testing/tests/DevExpress.ui.widgets/scrollable.tests.js

```diff
@@ -1911,6 +1890,37 @@ QUnit.test("content size should be rounded to prevent unexpected scrollbar appea
     assert.ok(scrollbar.$element().is(":hidden"), "scrollbar is not visible");
 });
 
+QUnit.module("Hoverable interaction", () => {
```

Comment:
These tests do not take into account the change of the "disabled" option in the runtime